docs/application/native/guides/app-management/TOC.md

```diff
@@ -11,6 +11,7 @@
 #### [Overview](/application/native/guides/app-management/dali-app.md)
```

Comment:
It looks like dali-widget-app.md page is added under dali-app.md. In **dali-app.md** page, there is a section which describes sub page, "Basic UI applicaion" and "Watch application", under title "Application Types'. Isn't this kind of description for "widget applicaion" needed?

Reply:
@mijin85cho done